CMakeLists.txt

```diff
@@ -325,6 +318,10 @@ endif(WITH_XIO)
 
 #option for RGW
 option(WITH_RADOSGW "Rados Gateway is enabled" ON)
+if(${WITH_RADOSGW})
```

Comment:
could you put `if(WITH_RADOSGW)` instead, just to be more consistent with other places where we check options. (yes, i know we use `if(${WITH_RADOSGW})` somewhere else. but we can take every chance to fix it)